build/root/WORKSPACE

```diff
@@ -38,15 +38,15 @@ http_archive(
 
 load("@bazel_skylib//:lib.bzl", "versions")
 
-versions.check(minimum_bazel_version = "0.10.0")
+versions.check(minimum_bazel_version = "0.10.1")
```

Comment:
is there a particular need for `0.10.1`?

Reply:
Oops. Just a case of version number blur when updating stuff.